invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/deposit.less

```diff
@@ -46,6 +47,13 @@
     font-size: 0.9em;
   }
 
+  .ui.form label.helptext{
```

Comment:
I think this rule can be merged with the above if not the above already applies...Can you double check?

Reply:
I pushed the change, how come its not ```outdated```?